lvmd/testutils/test_utils.go

```diff
@@ -1,4 +1,4 @@
-package lvmd
+package testutils
```

Comment:
The functions in this package seems to be only used in test codes.
How about moving to `utils_test.go` file without changing package rather than separating package?

Reply:
I tried to leave them where they were, but the compile was failing with ` "import cycle not allowed in test"`.  I tried a number of different things before I ended up moving to a separate package that was different for both consumers of them.

Reply:
Understood.